sensorfw.pro

```diff
@@ -64,6 +64,8 @@ contains(CONFIG,hybris) {
                adaptors
 } else {
     config_hybris {
+    # Reorder so that adaptors are built after hybris.
+    SUBDIRS -= adaptors
```

Comment:
Would it be cleaner to move the SUBDIRS += adaptors from near the beginning of the file to the end of the file and remove adding adaptors from both here and above in the contains part of the code?

Reply:
AFAICT this works / makes difference because there is "CONFIG += ordered" present -> fixes an issue -> already useful as it is.

But given that sensorfwd builds are painfully slow I started to wonder if it would be possible to drop that CONFIG=ordered bit, use explicit dependencies ("core/hybris.pro.depends += adaptors" or however it is done when a SUBDIRS element is not a plain subdirectory?) and thus enable parallel builds? And whether that would make any difference from build time pov?